### ISSUE TITLE
NoConsecutiveBlankLines rule does not work

### ISSUE THREAD

**Issue by weekens** [arturbosch/detekt#1553]
The `NoConsecutiveBlankLines` rule does not detect consecutive blank lines in Kotlin file.

## Expected Behavior
The `NoConsecutiveBlankLines` rule should detect and report consecutive blank lines in Kotlin files.

## Observed Behavior
Detekt check passes without failures on Kotlin files with several consecutive blank lines. The problem can be reproduced with default Detekt configuration where the rule is enabled.

## Steps to Reproduce
- create a valid Kotlin file with several consecutive blank lines
- run Detect with `--generate-config` option to generate default config; this config has `NoConsecutiveBlankLines` rule enabled; if it's not enabled, you'll need to enable it
- run Detect with `--config default-detekt-config.yml` option

You can use [this repository](https://github.com/weekens/test-detekt) to reproduce the problem. Just run the `reproduce.sh` script.

## Context
The problem prevents us from detecting blank line issues during manual and automatic build in Kotlin projects. We typically use Detect with Maven using [this plugin](https://github.com/Ozsie/detekt-maven-plugin), but the issue is with Detekt itself.

## Your Environment

* Version of detekt used: 1.0.0-RC14
* Operating System and version: Elementary OS Juno
* Link to your project (if it's a public repository): https://github.com/weekens/test-detekt

**Comment by arturbosch** [comment]
Hey, `NoConsecutiveBlankLines` is a rule coming from the `formatting` ruleset which will be enabled when adding `--plugins detekt-formatting.jar`, please also see https://arturbosch.github.io/detekt/#adding-more-rule-sets.
Does the maven plugin have support for additional detekt extensions?

**Comment by weekens** [comment]
Yup, that worked! Thanks for the answer!

For those who'll possibly come across this issue, here's the Maven configuration to enable this rule:

```xml
<plugin>
  <groupId>com.github.ozsie</groupId>
  <artifactId>detekt-maven-plugin</artifactId>
  <version>1.0.0.RC9.2</version>
  <dependencies>
    <dependency>
      <groupId>io.gitlab.arturbosch.detekt</groupId>
      <artifactId>detekt-formatting</artifactId>
      <version>1.0.0.RC9.2</version>
    </dependency>
  </dependencies>
  <executions>
    <execution>
      <phase>validate</phase>
      <goals><goal>check</goal></goals>
      <configuration>
        <config>detekt.yml</config>
      </configuration>
    </execution>
  </executions>
</plugin>
```
